acceptance/pe/setup/pre_suite/00_setup_env.rb

```diff
@@ -0,0 +1,60 @@
+require 'json'
+
+# Copies puppet ca from master to host
+#
+# === Returns
+#
+# +string+ - path of ca file or nil on fail
+def copy_ca_from_master_to(host)
+  ca_pem_contents = on(master, 'cat /etc/puppetlabs/puppet/ssl/certs/ca.pem').stdout.chomp
+  if (host.platform =~ /win/)
+    ca_pem_location       = ['C:', 'Windows', 'Temp', 'ca.pem'].join('\\')
+  else
+    ca_pem_location       = ['/tmp', 'ca.pem'].join('/')
+  end
+  res = create_remote_file(host, ca_pem_location, ca_pem_contents) unless master.hostname == host.hostname
+  return res.exit_code == 0 ? ca_pem_location : nil
+end
+
+step "Install Puppet Enterprise." do
+  install_pe
+end
+
+step 'copy ca.pem from master to client node' do
+  client = find_only_one('client')
+  copy_ca_from_master_to(client)
```

Comment:
the result of this call needs to set the value for `$ca_pem_location` to be consumed later.

i.e. should be
```
$ca_pem_location = copy_ca_from_master_to(client)
```